src/main/java/tutoraid/model/lesson/Lesson.java

```diff
@@ -63,7 +61,17 @@ public Price getPrice() {
     }
 
     public Students getStudents() {
-        return students;
+        if (students.isPresent()) {
```

Comment:
This is the problematic method that requires `allStudents` to be public and static, unless I find a way to pass the model to Lesson